west.yml

```diff
@@ -108,7 +108,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 3a5ee62b2d13d6acea9998ea32fde5a4176cafc0
+      revision: pull/634/head
```

Comment:
Please update SHA: `7588c863d46d35084463e7ee871d79b4ef935f85` and squash whatever needs to be squashed.